examples/01-filter/image-fft.py

```diff
@@ -0,0 +1,94 @@
+"""
+.. _image_fft_example:
+
+Fast Fourier Transform
+~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to apply a Fast Fourier Transform (FFT) to a
+:class:`pyvista.UniformGrid` using :func:`pyvista.UniformGridFilters.fft`
+filter.
+
+Here, we demonstrate FFT usage by denoising an image, effectively removing any
+"high frequency" content by performing a `low pass filter
+<https://en.wikipedia.org/wiki/Low-pass_filter>`_
+
+This example was inspired by `Image denoising by FFT
+<https://scipy-lectures.org/intro/scipy/auto_examples/solutions/plot_fft_image_denoise.html>`_.
+
+"""
+
+import pyvista as pv
+from pyvista import examples
+
+###############################################################################
+# Load the example Moon-landing image and plot it.
+
+image = examples.download_moonlanding_image()
+print(image.point_data)
+
+# Create a theme that we can reuse when plotting the image
+grey_theme = pv.themes.DocumentTheme()
+grey_theme.cmap = 'gray'
+grey_theme.show_scalar_bar = False
+grey_theme.axes.show = False
+image.plot(theme=grey_theme, cpos='xy', text='Unprocessed Moon Landing Image')
+
+###############################################################################
+# Apply FFT to the image
+#
```

Comment:
(Only additional remark after having looked at doc build)

There are a few heading-looking lines in this example that aren't marked up as headings:
![Screenshot from 2022-06-05 22-57-59](https://user-images.githubusercontent.com/17914410/172070406-3486d71a-7795-45cf-b79b-fa7313bfbc02.png)

I'd be tempted to use some kind of subheading markup for these, but both `~~~~~` and `-----` underlining renders with way too large titles (based on other examples). If we keep these as regular text, they'll need periods at the end.